src/renderer/api/json-api.ts

```diff
@@ -92,28 +92,27 @@ export class JsonApi<D = JsonApiData, P extends JsonApiParams = JsonApiParams> {
     });
   }
 
-  protected parseResponse<D>(res: Response, log: JsonApiLog): Promise<D> {
+  protected async parseResponse<D>(res: Response, log: JsonApiLog): Promise<D> {
```

Comment:
Is this fixing/changing something related to this PR?

Reply:
No... I probably should just remove this file from this PR then. Though this code is clearer and uses the `async` style, so maybe it is fine.

Reply:
PR should have only changes related to it's scope, let's remove this from here. (this applies also to other non-related changes).